rfcs/formats/assets/transfer.md

```diff
@@ -10,5 +10,44 @@
 - Status: Proposed
 - Since: 02-Apr-2022
 
-## Summary
```

Comment:
I think you should retain the "Summary" heading, to be consistent with all our other RFCs, where there's no text directly after the header.

Reply:
Done